src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -60,22 +60,22 @@ const Embedding = (props) => {
 
   const focusData = useSelector((state) => state.cellInfo.focus);
   const focusedExpression = useSelector((state) => state.genes.expression.data[focusData.key]);
+  const cellSets = useSelector(getCellSets());
   const {
     properties: cellSetProperties,
     hierarchy: cellSetHierarchy,
     hidden: cellSetHidden,
-  } = useSelector(getCellSets());
+  } = cellSets;
```

Comment:
Why not use the old style?

Reply:
Because I need to pass the whole cellSets to another function